### PR TITLE
Fix CV numpy stuff

### PR DESCRIPTION
**Pull request by aidanwoodcu**
## Changes

- Makes the pointing vectors produced by `cv.py` and `process_frame.py` actually the correct type/shape. This should fix [the bug Lyon saw here](https://discord.com/channels/1149059111615471646/1159562394926850228/1212542070762377256).

I've tried to be as minimally intrusive as possible with these changes but a lot of this code could use a clean up/refactor.

### CODE REVIEW

File: FSW/config/constants.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
 ## TODO Update the formatting of the camera extrinsic parameters
 ## TODO Configure a way to get the camera extrinsic parameters accurately
 EXTRINSICS_PI_CAMERA_DCM: npt.NDArray = np.array([[1.0, 0.0, 0.0], [0.0, 1.0, 0.0], [0.0, 0.0, 1.0]]) #DCM From UAS 2 Camera
-EXTRINSICS_PI_CAMERA_TVEC: npt.NDArray = np.array([0.0, 0.0, 0.0]) #tvec for Camera from UAS in UAS Frame
+EXTRINSICS_PI_CAMERA_TVEC: npt.NDArray = np.array([[0.0], [0.0], [0.0]]) #tvec for Camera from UAS in UAS Frame
```
Comment (aidanwoodcu) [inline FSW/config/constants.py:42]: Fixed orientation of vector (previously 1x3 + 3x1 gave 3x3). This is probably a bad way to do this but numpy is gross.

File: FSW/functional/process_frame.py [hunk 2]
```diff
@@ -145,7 +144,7 @@ def camera_frame_to_UAS_frame(position: np.ndarray) -> np.ndarray:
     ## TODO Make sure the matrix mult is right
 
     #Convert from rotation Vector to Rotation Matrix
-    position_UASFrame = constants.EXTRINSICS_PI_CAMERA_DCM * (position) + constants.EXTRINSICS_PI_CAMERA_TVEC
+    position_UASFrame = np.matmul(constants.EXTRINSICS_PI_CAMERA_DCM, position) + constants.EXTRINSICS_PI_CAMERA_TVEC
```
Comment (aidanwoodcu) [inline FSW/functional/process_frame.py:147]: Do actual matrix multiplication because numpy doesn't like *

File: FSW/config/constants.py (hunk 1 above)
Comment (Tim-Behrer) [reply FSW/config/constants.py:42]: I can take a look at the nitty-gritty on this stuff and see if there is a more efficient way to do this if necessary.

File: FSW/cv.py [hunk 3]
```diff
@@ -75,17 +75,20 @@ def frame_callback(msg: Image):
     for id in detection_info.ids:
         ##If condition to check for valid ID
         if constants.ARUCO_ID2RGV_DICT.__contains__(id):
+            rgv_id = constants.ARUCO_ID2RGV_DICT[id]
             ##Recent Sighting Message Definition
             sighting = RecentSighting()
             sighting.timestamp = rospy.Time.now()
-            sighting.rgv_id = constants.ARUCO_ID2RGV_DICT[id]
```
[the remark this answers is not in the document]
Comment (aidanwoodcu) [reply FSW/cv.py:81]: I filled out the `rgv_id` field in `direction_vector_msg` which has the same value so I just pulled out the dictionary search .

**Review by LyonFoster — APPROVED** [review]

**Pull request merged**